tools/scylla-nodetool.cc

```diff
@@ -405,6 +406,49 @@ void describering_operation(scylla_rest_client& client, const bpo::variables_map
     }
 }
 
+void describecluster_operation(scylla_rest_client& client, const bpo::variables_map&) {
+    const auto cluster_name_res = client.get("/storage_service/cluster_name");
+    const auto snitch_name_res = client.get("/snitch/name");
+    // ScyllaDB does not support Dynamic Snitching, and will not.
+    // see https://github.com/scylladb/scylladb/issues/208
+    const auto dynamic_snitch = "disabled";
+    const auto partitioner_name_res = client.get("/storage_service/partitioner_name");
+    fmt::print("Cluster Information:\n"
+               "\tName: {}\n"
+               "\tSnitch: {}\n"
+               "\tDynamicEndPointSnitch: {}\n"
+               "\tPartitioner: {}\n",
+               rjson::to_string_view(cluster_name_res),
+               rjson::to_string_view(snitch_name_res),
+               dynamic_snitch,
+               rjson::to_string_view(partitioner_name_res));
+
+    const auto schema_versions = client.get("/storage_proxy/schema_versions");
+    fmt::print("\tSchema versions:");
+    if (!schema_versions.IsArray()) {
```

Comment:
These kinds of checks are not required (aplies to similar ones below).
ScyllaDB is unlikely to change its JSON and even if it does, `rjson` will already throw an error if the type of some object is not the expected one (e.g. `GetArray()` throws an exception if current object is not an array).

Reply:
> These kinds of checks are not required (aplies to similar ones below). ScyllaDB is unlikely to change its JSON and even if it does, `rjson` will already throw an error if the type of some object is not the expected one (e.g. `GetArray()` throws an exception if current object is not an array).

thanks. i failed to check if we define `RAPIDJSON_ASSERT()`.